docs/conceptual/list.chunkbysize['t]-function-[fsharp].md

```diff
@@ -0,0 +1,80 @@
+---
+title: List.chunkBySize<'T> Function (F#)
+description: List.chunkBySize<'T> Function (F#)
+keywords: visual f#, f#, functional programming
+author: banashek
+---
+
+# List.chunkBySize<'T> Function (F#)
+
+Divides the input list into chunks of size at most chunkSize.
+
+**Namespace/Module Path:** Microsoft.FSharp.Collections.List
+
+**Assembly:** FSharp.Core (in FSharp.Core.dll)
+
+
+## Syntax
+
+```fsharp
+// Signature:
+List.chunkBySize : int -> 'T list -> 'T list list
+
+// Usage:
+List.chunkBySize chunkSize list
+```
+
+#### Parameters
+*chunkSize*
+Type: [int](https://msdn.microsoft.com/library/025d5455-3622-4ea5-9573-3ecbd4ee1375)
+
```

Comment:
I added single line breaks before and after each heading, and reduced the double breaks to single breaks.